### PR TITLE
[doc] add runtime system doc

### PR DESCRIPTION
**Pull request by robin1001**

### CODE REVIEW

File: runtime/README.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# Runtime on WeNet
+
+This is the runtime of WeNet.
+
+We are going to support the following platforms:
+
+1. Various deep learning engines, such as LibTorch, ONNX, OpenVINO, TVM and so on.
```
Comment (pengzhendong) [inline runtime/README.md:7]: deep learning inference engines

File: runtime/README.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+2. Various OS, such as android, iOS, Harmony and so on.
+3. Various AI chips, such as GPU, Horzion BPU and so on.
+4. Various hardware platforms, such as Rasphberry Pi.
```
Comment (pengzhendong) [inline runtime/README.md:10]: Raspberry

File: runtime/README.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+5. Various language bingding, such as python and go.
+
+Feel free to volunteer yourself if you are interested in trying out some items(they do not have to be on the list).
+
+## Progress
+
+For each platform, we will create a subdirectory on in runtime. Currently, we have:
+
+- [x] LibTorch: in c++, the default engine of WeNet.
+- [x] OnnxRuntime: in c++, the offical runtime for onnx model.
```
Comment (pengzhendong) [inline runtime/README.md:20]: official

File: runtime/README.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+- [x] GPU: in python, powered by triton.
+- [x] android: in java, it shows an APP demo.
+- [ ] Language bingding
```
Comment (pengzhendong) [inline runtime/README.md:23]: binding

**Comment by robin1001** [comment]
列文胡克啊

File: runtime/README.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+  - [x] bingding/python: python is the first class for binding.
+  - [ ] bingding/go: ongoing.
+
+
+Please note `server` and `device` is depreated, it will be removed in the future.
```
Comment (pengzhendong) [inline runtime/README.md:28]: deprecated

File: runtime/README.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+## Progress
+
+For each platform, we will create a subdirectory on in runtime. Currently, we have:
+
+- [x] LibTorch: in c++, the default engine of WeNet.
+- [x] OnnxRuntime: in c++, the offical runtime for onnx model.
+- [x] GPU: in python, powered by triton.
+- [x] android: in java, it shows an APP demo.
+- [ ] Language bingding
+  - [x] bingding/python: python is the first class for binding.
```
Comment (pengzhendong) [inline runtime/README.md:24]: binding

File: runtime/README.md [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+  - [ ] bingding/go: ongoing.
```
Comment (pengzhendong) [inline runtime/README.md:25]: binding

File: runtime/README.md [hunk 8]
```diff
@@ -0,0 +1,28 @@
...
+2. Various OS, such as android, iOS, Harmony and so on.
+3. Various AI chips, such as GPU, Horzion BPU and so on.
+4. Various hardware platforms, such as Rasphberry Pi.
+5. Various language bingding, such as python and go.
+
+Feel free to volunteer yourself if you are interested in trying out some items(they do not have to be on the list).
+
+## Progress
+
+For each platform, we will create a subdirectory on in runtime. Currently, we have:
```
Comment (pengzhendong) [inline runtime/README.md:17]: subdirectory in runtime

File: runtime/README.md [hunk 9]
```diff
@@ -0,0 +1,28 @@
...
+
+This is the runtime of WeNet.
+
+We are going to support the following platforms:
+
+1. Various deep learning engines, such as LibTorch, ONNX, OpenVINO, TVM and so on.
+2. Various OS, such as android, iOS, Harmony and so on.
+3. Various AI chips, such as GPU, Horzion BPU and so on.
+4. Various hardware platforms, such as Rasphberry Pi.
+5. Various language bingding, such as python and go.
```
Comment (pengzhendong) [inline runtime/README.md:11]: binding

File: runtime/README.md [hunk 10]
```diff
@@ -0,0 +1,28 @@
+# Runtime on WeNet
+
+This is the runtime of WeNet.
+
+We are going to support the following platforms:
+
+1. Various deep learning inference engines, such as LibTorch, ONNX, OpenVINO, TVM and so on.
```
Comment (pengzhendong) [inline runtime/README.md:7]: Consider inserting a comma before `and so on`.

File: runtime/README.md (hunk 5 above)
Comment (pengzhendong) [reply runtime/README.md:28]: Plz fix this one.

**Review by pengzhendong — APPROVED** [review]

**Pull request merged**